### PR TITLE
Fixes SiCKRAGETV/sickrage-issues/issues/2954

### PR DESCRIPTION
**Pull request by miigotu**

### CODE REVIEW

**Comment by MGaetan89** [comment]
What do the `[]` you added?

**Comment by miigotu** [comment]
It is the same as adding a back tick, it makes sure sql parses it as a string and not an int

File: sickbeard/db.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def tableInfo(self, tableName):
         :param tableName: name of table
         :return: array of name/type info
         """
-        sqlResult = self.select("PRAGMA table_info(%s)" % tableName)
+        sqlResult = self.select("PRAGMA table_info(`%s`)" % tableName)
```
Comment (miigotu) [inline sickbeard/db.py]: @MGaetan89 this does the same thing

**Comment by MGaetan89** [comment]
Ok, I didn't know the `[]` solution, only the back ticks. Any reason mix the two in this PR?

**Comment by miigotu** [comment]
Well, I originally had all backticks, but everywhere else in SR uses the brackets. brackets in table_info just feels retarded to me so I left that one backticks.

**Comment by MGaetan89** [comment]
I understand. Thanks for the info. It's good to learn something new everyday :)

**Pull request merged**